cra/task-bom.yaml

```diff
@@ -284,6 +284,17 @@ spec:
           #!/bin/sh
           source /steps/next-step-env.properties
 
+          if [[ "$(params.repository)" == *"git.cloud.ibm.com"* ]] && [[ "$(params.scm-type)" != "gitlab" ]]; then
```

Comment:
Since this check is needed in multiple tasks, can we turn this code into a reusable script?